main.go

```diff
@@ -97,7 +132,71 @@ func canStartWarehouse() bool {
 	return warehouseMode != config.OffMode
 }
 
+func runAllInit() {
```

Comment:
We are doing as a temporary work around, since we need to control when the init method which are parsing the configuration is called.

Otherwise we would be unable to override default configuration from the testing code.